pmm-app/src/pmm-settings/Parts/AlertManager/AlertManager.test.tsx

```diff
@@ -8,7 +8,7 @@ describe('Settings Part test', () => {
     const component = renderer.create(
       <AlertManager
         settings={{
-          alert_manager_ip: '192.168.0.1',
+          alert_manager_url: '192.168.0.1',
```

Comment:
That's not an URL. It will not work. Please actually test it.

Reply:
Updated it now, I'll also change this tests, because now this fields is not really matters, we don't check it